src/Driver/driver_configs.jl

```diff
@@ -252,12 +259,24 @@ function AtmosLESConfiguration(
         meshwarp = meshwarp,
     )
 
+    if cutofforder_horz < polyorder_horz || cutofforder_vert < polyorder_vert
+        filter = CutoffFilter(
+            grid,
+            (cutofforder_horz + 1, cutofforder_horz + 1, cutofforder_vert + 1),
+        )
+    else
+        filter = nothing
+    end
+
+
     @info @sprintf(
         """
 Establishing Atmos LES configuration for %s
     precision               = %s
     horiz polynomial order  = %d
     vert polynomial order   = %d
+    cutofforder_horz        = %d
+    cutofforder_vert        = %d
```

Comment:
for consistency with the other prints, I'd use something like:
```suggestion
    horiz cutoff order        = %d
    vert cutoff order          = %d
```